output.tf

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
remove the blank file?
We should try to use more of data resources instead of outputs where possible.